### DISCUSSION TITLE
idle_power and throttle_idle

### DISCUSSION THREAD

**Discussion by Colechr56** [iNavFlight/inav#10124] [Support] [answered]
I have an f405wingApp, with an x8r receiver, and BZ-251 gps. This is all to go in a boat with twin screw, bow thruster, and twin rudders. Mostly I have followed the instructions and setup, but have an issue with the Arming Flags. During the setups, the flags showed Throttle Arming flags, among the Arming switch flags. But eventually it seems to resolve to a situation where if the Motor Idle Power in Outputs is set at zero, the Throttle arming flag is on. If I set the Motor Idle Power to 2%, then this flag clears, and the FC can be armed. I feel that I don't want the value at 15% as with centre position 1500microseconds engines off, would be an issue? I feel I may be misunderstanding this?
Additionally in a CLi dump I found that there is a set idle_power = 0, and a set throttle_idle = 2.000. The last equates to the Motor idle power? What is the other for?
Can you advise more reading directions?
regards Chris C

**Comment by breadoven** [comment, marked as answer]
`idle_power` is related to the RTH estimated time remaining function https://github.com/iNavFlight/inav/blob/master/docs/Battery.md#remaining-flight-time-and-flight-distance-estimation.

**Reply by Colechr56** [reply]
Many thanks, I had missed that in searches.
